### DISCUSSION TITLE
Cannot add two decimals on number collection type

### DISCUSSION THREAD

**Discussion by ssayanm** [strapi/strapi#7396] [Legacy Topics]
Hi @lauriejim , 

I am facing an issue.

Suppose I choose a collection type as "number" and then "Number format" as either decimal or float and then I try to add a number "129.12" and Strapi is not allowing me to do it I can only add "129.1" or "129.2"

I am using Strapi v3.1.3

[Screenshot1](https://u.pcloud.link/publink/show?code=XZo52vkZnE9KRVEGNLunJpmtYr23qSPrqFAy)
[Screenshot2](https://u.pcloud.link/publink/show?code=XZLJ2vkZkWB28T5cOjYyo0bVnfPytjKnOIRk)

Thank you

**Comment by derrickmehaffy** [comment]
@ssayanm please open up a bug report and follow the bug report template so we can properly address this issue.

**Reply by ssayanm** [reply]
@derrickmehaffy thank you.. created a bug report

**Reply by derrickmehaffy** [reply]
Thank you!
